.github/workflows/log-viewer-docs.yaml

```diff
@@ -2,6 +2,8 @@ name: "docs"
 
 on:
   pull_request:
```

Comment:
if we just alter the behaviour for PRs (rather than also `push`), we should mention that in this PR title.